lib/gollum-lib/filter/tags.rb

```diff
@@ -246,12 +246,7 @@ def process_external_link_tag(tag)
   #
   # Returns the String HTML if the tag is a valid file link tag or nil
   #   if it is not.
-  def process_file_link_tag(tag)
-    parts = tag.split('|')
-    return if parts.size.zero?
-
-    name = parts[0].strip
-    path = parts[1] && parts[1].strip
+  def process_file_link_tag(name, path)
     if path && file = @markup.find_file(path)
       path = ::File.join @markup.wiki.base_path, file.path
```

Comment:
This methods returns HTML. Is there any way to refactor that so that gollum (and not gollum-lib) wraps `path` and `name` in the appropriate html tags?